src/commonMain/kotlin/exchange.dydx.abacus/utils/TradeValidationTracker.kt

```diff
@@ -14,7 +15,8 @@ class TradeValidationTracker(
     fun logValidationResult(payload: TradeValidationPayload) {
         if (payload == lastSeen) return
 
-        trackingProtocol?.log(AnalyticsEvent.TradeValidation.name, TrackingJson.encodeToString(TradeValidationPayload.serializer(), payload))
```

Comment:
what was the issue with the `serializer`?

Reply:
No.. it's the in-line call.  Breaking it down to two lines fixed the problem.... It's definite a bug with code optimization.